writing-guide/readme.md

```diff
@@ -59,6 +59,10 @@ Always write in an active writing style. [Other styles can be effective](https:/
 
 Avoid scarcely-used words unless there is no substitution. In this case, always explain the word. In most cases, however, complex things can and should be expressed simply.
 
+## Find a suitable ending
+
+Don't leave the reader hanging right at the end. In the final paragraph, summarize the article. Go back to the beginning and the goal of the article; if you posed a question, make sure that question is answered. If you made a statement, make sure you summarize the various arguments you discussed along the way. And just like the beginning, try to spark the interest of the reader by leaving them with some food for thought.
+
```

Comment:
Awesome!

"Go back to the beginning and the goal of the article"
Is a bit weird. 'Go back' to the beginning could be removed because you don't necessarily want to people to think about the beginning. It's about the points that were made in the entire article. (could actually remove the whole "Go back to the beginning and the goal of the article; " bit)

"question is answered."
This is often also done with various arguments and can the sentences can therefore probably be combined. (what do you think?)
"And just like the beginning, try to spark the interest of the reader by leaving them with some food for thought."
The reason for sparking the interest is different though (not sure if useful to mention ;)